ci/templates/maven-test-project/pom.xml

```diff
@@ -0,0 +1,189 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>org.example</groupId>
+    <artifactId>maven-test-project</artifactId>
+    <version>1.0-SNAPSHOT</version>
+
+    <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <kotlin.code.style>official</kotlin.code.style>
+        <kotlin.compiler.jvmTarget>1.8</kotlin.compiler.jvmTarget>
+        <kotlin.version>2.0.21</kotlin.version>
```

Comment:
What do you think about using kotlin 2.1.0?  

As I see that on CI, the script doesn't pass a overriden value:
```
mvn install exec:java -Dexec.mainClass="MainKt" -Dcompose.version=%version.COMPOSE%
```

Reply:
Changed. I will pass Kotlin version on CI later. This is also needed for the other templates

Reply:
https://youtrack.jetbrains.com/issue/CMP-7456/Check-Templates-Examples-on-CI-with-multiple-Kotlins